deformable_ompl/PlanPathQuery query
---
deformable_ompl/PlanPathResult result
